plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -81,13 +73,16 @@ public void setExtensions()
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
-            case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY:
+            case SUPPORTS_JOIN_PUSHDOWN_WITH_VARCHAR_INEQUALITY:
```

Comment:
This is false only because you haven't changed the `implementJoin` code? Correct? We can possibly use the same technique to allow joins on varchars to get pushed down?

Asking because I'd like to submit a follow-up PR for the join pushdown case.

Reply:
Yes. This will be unnecessary if we implement join pushdown for VARCHAR columns.